File: BackEnd/src/main/java/com/portfolio/demo/Entity/Servicio.java

package com.portfolio.demo.Entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;


@Getter @Setter
@Entity
public class Servicio {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    
    private String nombreS;
    private String descripcionS;

    public Servicio() {
    }

    public Servicio(String nombreS, String descripcionS) {
        this.nombreS = nombreS;
        this.descripcionS = descripcionS;
    }
    
}

File: src/app/model/servicio.ts
export class Servicio {
    id?: number;
    nombreS: string;
    descripcionS: string;
    imageS: string;

    constructor(nombreS: string, descripcionS: string, imageS: string) {
        this.nombreS = nombreS;
        this.descripcionS = descripcionS;
        this.imageS = imageS;
    }

}
